src/main/java/uk/gov/hmcts/darts/arm/service/ExternalObjectDirectoryService.java

```diff
@@ -2,11 +2,13 @@
 
 import org.springframework.data.domain.Pageable;
 import uk.gov.hmcts.darts.common.entity.ExternalObjectDirectoryEntity;
+import uk.gov.hmcts.darts.common.entity.MediaEntity;
 
 import java.util.List;
 
 public interface ExternalObjectDirectoryService {
 
     List<ExternalObjectDirectoryEntity> findFailedStillRetriableArmEods(Pageable pageable);
 
+    boolean hasNotAllMediaBeenCopiedFromInboundStorageForAtsProcessing(List<MediaEntity> mediaRequestFiles);
```

Comment:
mediaRequestFiles seems like wrong name for list of mediaEntity

Reply:
`mediaEntities``